erpnext/vehicles/page/workshop_cp/workshop_cp.js

```diff
@@ -233,6 +288,223 @@ class WorkshopCP {
 			d.show();
 	}
 
+	assign_technician(e) {
+		let task = $(e.target).attr('data-task');
+		let task_data = this.get_row_data("Task", task);
+		var d = new frappe.ui.Dialog({
+			title: __('Assign Technician'),
+			fields: [
+				{
+					"label": "Task",
+					"fieldname": "name",
+					"fieldtype": "Link",
+					"options": "Task",
+					"default": task,
+					"read_only": 1,
+					"reqd": 1,
+				},
+				{
+					"label" : "Subject",
+					"fieldname": "subject",
+					"fieldtype": "Data",
+					"default": task_data.subject,
+					"read_only": 1,
+				},
+				{
+					"label" : "Technician",
+					"fieldname": "employee",
+					"fieldtype": "Link",
+					"options": "Employee",
+					"onchange": () => {
+						let employee = d.get_value('employee');
+						if (employee) {
```

Comment:
remove conditional blocks. frappe.db.get_value will handle the nonexistent employee case